Automattic/wc-calypso-bridge
# Fix JS unit tests

### Changes proposed in this Pull Request:



Fixes the broken JS unit tests.

- Add jest and babel configs to add necessary mocks and setup files in the bridge.
- Add `@testing-library/jest-dom` and `@testing-library/react` libs as we use them in our tests.
- Add test mock and setup files
- Fix wrong `enqueueScript` import path in payment gateway test






### How to test the changes in this Pull Request:




1. Run `nvm use` - if you don't have version 16 of node installed, you may need to run nvm install 16
2. Run `npm install`
3. Run `npm run test:unit`
4. Confirm all tests pass
5. Run `npm run build`
6. Verify that the build completes successfully

![Screenshot 2023-04-25 at 11 57 12](https://user-images.githubusercontent.com/4344253/234171023-66b8d77e-f2e4-4a2a-856f-f2f6cc7d3445.png)




### Other information:




### FOR PR REVIEWER ONLY:
